### ISSUE TITLE
Segmentation Fault during ABI Generation 

### ISSUE THREAD

**Issue by jchung00** [EOSIO/eosio.cdt#672]
Seems like the code compiles but the ABI is not generated using `eosio-cpp`. I'm guessing that the vector of tuples is causing this problem.
```
struct [[eosio::table]] stake {
        name staker;
        vector<tuple<uint64_t, asset, bool>> stakelist;

        uint64_t primary_key() const { return staker.value; }

        EOSLIB_SERIALIZE( stake, (staker)(stakelist) )
};
```
Output:
```
Warning, empty ricardian clause file
Warning, empty ricardian clause file
Segmentation fault: 11
```

**Comment by larryk85** [comment]
This should be resolved as of v1.6.3.

**Comment by jeffreyssmith2nd** [comment]
Confirmed resolved on 1.6.3.
